COMPATIBILITY.md

```diff
@@ -150,7 +150,8 @@ For cases when a first-class Pipeline step (rather than an adaptation of functio
 - [X] `mailer`: `mail` step in Pipeline 1.2
 - [ ] `artifactory`: [JENKINS-30121](https://issues.jenkins-ci.org/browse/JENKINS-30121)
 - [X] `email-ext`: `emailext` step since 2.41
-
+- [X] `MarathonStep` (`marathon`): step since 1.2.1
+- 
```

Comment:
Gratuitous trailing space seems to break formatting badly. Click the _Display the rich diff_ button in the _Files changed_ tab to see what I mean.